### ISSUE TITLE
Can't unPAUSE DOSBox-X (with C_DEBUG 0)

### ISSUE THREAD

**Issue by emendelson** [joncampbell123/dosbox-x#632]
I built the latest code with C_DEBUG 0 so that I could experiment with pausing a game. Alt-Pause correctly pauses DOSBox-X, but a second press of Alt-Pause seems not to un-pause it. I can't type anything further into the command line.

I may be misunderstanding something, so apologies if this is a waste of bandwidth.

**Comment by emendelson** [comment]
That's exactly what I did. I pressed Alt-Pause (and DOSBox-X paused) then I lifted my hands from the keyboard, tried to type a few letters (nothing happened, of course), then pressed Alt-Pause again. The PAUSED string disappeared from the titlebar but DOSBox-X remained paused.

I seem to be able to unpause DOSBox-X with a sequence like this:

Alt-Pause
[release keys, try to type]
Esc [PAUSED string disappears, still can't type]
Alt-Pause

In other words, unpausing seems to need Esc followed by Alt-Pause, even though the PAUSED string disappears (prematurely) when I press Esc.

**Comment by joncampbell123** [comment]
Try tapping the CTRL + SHIFT + ALT keys after un-pausing. It's possible the DOS environment still thinks those keys are down.

**Comment by emendelson** [comment]
That solves it! Is it possible to do something in the program itself that would accomplish the same thing?

**Comment by joncampbell123** [comment]
I think so. I was able to fix CTRL+SHIFT+ALT status when leaving the mapper interface, perhaps the same fix could be applied to Alt+PAUSE as well.

**Comment by emendelson** [comment]
That's terrific - thank you.

**Comment by joncampbell123** [comment]
Fixed

**Comment by emendelson** [comment]
And thank you again. (And, at the risk of sounding selfish, I hope you won't forget  https://github.com/joncampbell123/dosbox-x/issues/490 )
